cve_bin_tool/output_engine/html_reports/templates/row_product.html

```diff
@@ -13,7 +13,7 @@
         </div>
 
         <div class="col-sm-2">
-            <span class="badge badge-pill badge-primary">{{ cve_count }}</span>
+        <span class="badge badge-pill badge-primary cve-count" total-cve-count={{ cve_count }}>{{ cve_count }}</span>
```

Comment:
Don't think we want to take this indent out, since it looks like we indent inside a div elsewhere.

Reply:
my bad